src/commandline_content.ts

```diff
@@ -48,10 +48,14 @@ export function resize() {
 }
 
 export function hide(){
-    cmdline_iframe.setAttribute("style", "display: none")
+    const height = '0px'
+    cmdline_iframe.setAttribute("style", `position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: ${height}; border: 0; padding: 0; margin: 0;`);
```

Comment:
Set height to 0px to hide cmdline, not sure if this cause bug.

Reply:
We should probably use `cmdline_iframe.style.cssText = "blah"` rather than setAttr: it's a bit less verbose.

Why have you made height a const?

Reply:
Just easy to review, if this is ok, I think we'd better move the style into css, and use class instead.